EIPS/eip-4834.md

```diff
@@ -4,27 +4,26 @@ title: Hierarchical Domains
 description: Extremely generic name resolution
 author: Pandapip1 (@Pandapip1)
 discussions-to: https://ethereum-magicians.org/t/erc-4834-hierarchical-domains-standard/8388
-status: Last Call
-last-call-deadline: 2022-08-31
+status: Final
 type: Standards Track
 category: ERC
 created: 2022-02-22
-requires: 20, 137, 165, 173
+requires: 137, 165
```

Comment:
I wonder if it needs to put 137 as required...

Reply:
It references EIP-137, and as per my own policy, I'll keep it as required.

Reply:
Will remove it if another editor would rather it not be required.

Reply:
I don't have a strong preference but I feel "requires" is confusing. In another Pull Request a comment from either you or @SamWilsn says if a referenced EIP is being referenced in Specification section, put it in the`requires`, otherwise don't put it. I think this is a good test. Since this EIP doesn't reference 137 in its Spec, I think in consistency to that policy, it can be removed.

Reply:
Micah isn't an editor anymore, for better or for worse :/